terraform/iam.tf

```diff
@@ -0,0 +1,38 @@
+data "aws_iam_policy_document" "secrets" {
+  statement {
+    effect = "Allow"
+    actions = [
+      "secretsmanager:GetResourcePolicy",
+      "secretsmanager:GetSecretValue",
+      "secretsmanager:DescribeSecret",
+      "secretsmanager:ListSecretVersionIds"
+    ]
+    resources = [
+      data.aws_secretsmanager_secret.admin_secret_key_base,
+      data.aws_secretsmanager_secret.admin_bearer_token,
+      data.aws_secretsmanager_secret.admin_oauth_id,
+      data.aws_secretsmanager_secret.admin_oauth_secret
+    ]
+  }
+
+  statement {
+    effect = "Allow"
+    actions = [
+      "kms:Encrypt",
+      "kms:Decrypt",
+      "kms:ReEncryptFrom",
+      "kms:ReEncryptTo",
+      "kms:GenerateDataKeyPair",
+      "kms:GenerateDataKeyPairWithoutPlainText",
+      "kms:GenerateDataKeyWithoutPlaintext"
+    ]
+    resources = [
+      data.aws_kms_key.secretsmanager_key.arn
+    ]
+  }
+}
+
+resource "aws_iam_policy" "secrets" {
```

Comment:
Do we want a policy to get a ECS session?

Reply:
We can add the policy in since we've already got it on the backend, sure thing. Will be deploying in a second PR anyway